src/backend/distributed/commands/alter_table.c

```diff
@@ -0,0 +1,1344 @@
+/*-------------------------------------------------------------------------
+ *
+ * alter_table.c
+ *	  Routines relation to the altering of tables.
+ *
```

Comment:
Given the size of this file, it would be nice to have some more description of the high-level steps of the later operations.